examples/reference/ambientLight/ambientLight0/.test.yml

```diff
@@ -0,0 +1,2 @@
+test:
+  reference: https://processing.org/reference/images/ambientLight_0.png
```

Comment:
I took another look at the original Processing reference and it is not consistent either -- some end in _, some end in _0, and some have numbers in the middle of the name... some are 0-indexed and some are (implicitly) 1-indexed.

-  https://processing.org/reference/images/

For example, the reference images for `arc()` are arc_.png, arc_2.png, and arc_3.png.

Given that the original material isn't consistent we don't need to worry right now about the new material being consistent -- it won't help us automate.